tests/console/console.go

```diff
@@ -17,7 +17,7 @@
  *
  */
 
-package tests
+package console
```

Comment:
We don't follow the `libxyz` pattern ? I am ok with that but feels weird with the others.

Reply:
Let's keep `console`